### PR TITLE
Openrpc doc

### PR DESCRIPTION
**Pull request by illuque-iov**
Added OpenRPC doc for our RPC methods:
- individual file for each method
- individual file for each schema
- individual file for each contendDescriptor

These individual files can be collected into a single file, the final OpenRPC doc, using `GenerateOpenRpcDoc` CLI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by rmoreliovlabs — APPROVED** [review]
Wow amazing!

File: doc/rpc/components/schemas/Transaction.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "Transaction": {
```
Comment (bguiz) [inline doc/rpc/components/schemas/Transaction.json:2]: I think we likely will need `Transaction` and `TransactionRemasc`, to take into account that the 3 `v,r,s` fields are null/undefined/zero in this RSK-specific type of tx

File: doc/rpc/components/schemas/BlockExampleResultFullTransaction.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      "extraData": "0x",
+      "size": "0xafe",
+      "gasLimit": "0x67c280",
+      "gasUsed": "0x0",
+      "timestamp": "0x5d1f7c06",
+      "transactions": [
+        {
+          "$ref": "#/components/schemas/TransactionExampleResult"
+        }
+      ],
```
Comment (bguiz) [inline doc/rpc/components/schemas/BlockExampleResultFullTransaction.json:25]: ... and following on from prev comment, this should have 1 `TransactionRemascExampleResult` followed by a `TransactionExampleResult`

Comment (illuque-iov) [reply doc/rpc/components/schemas/BlockExampleResultFullTransaction.json:25]: Sure, as soon as the task related to the Remasc fix gets implemented (it's WIP now), its related OpenRPC doc will be updated accordingly.

**Comment by illuque-iov** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
@illuque-iov , do we need `NOT_IMPLEMENTED` folder with its content? Seems majority of methods in there are not even part of the standard

**Review by Vovchyk — APPROVED** [review]

File: rskj-core/src/main/java/co/rsk/cli/tools/GenerateOpenRpcDoc.java [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+ * <br>
+ * <br>
+ * The <b>template</b> file contains basic and static OpenRPC info.
+ * <p>The individual json files contain the different <b>methods</b>, <b>schemas</b> and
+ * <b>contentDescriptors</b> (as per OpenRPC definition) which are maintained by devs to document implemented RPC methods
+ * <br>
+ * <br>
+ * Required cli args:
+ * <ol>
+ *     <li><b>args[0]</b>: work directory where the json template and individual json files are present</li>
```
Comment (VaWheel) [inline rskj-core/src/main/java/co/rsk/cli/tools/GenerateOpenRpcDoc.java:55]: due to the code 3 arguments are read. should we update this?

Comment (illuque-iov) [reply rskj-core/src/main/java/co/rsk/cli/tools/GenerateOpenRpcDoc.java]: (apologies, I was on vacation)

Good catch @VaWheel, added this field in the last minute and forgot to update the javadoc. Updated.

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by illuque-iov** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by illuque-iov** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Review by Vovchyk — APPROVED** [review]

**Pull request merged**